core/vtk/ttkJacobiSet/ttkJacobiSet.h

```diff
@@ -19,6 +19,24 @@
 /// This filter can be used as any other VTK filter (for instance, by using the
 /// sequence of calls SetInputData(), Update(), GetOutput()).
 ///
+/// The input data arrays needs to be specified via the standard VTK call
```

Comment:
I'll go ahead and merge that PR, but would it be possible to have exactly the same documentation for core/vtk/ttkReebSpace/ttkReebSpace.h?

Reply:
I'll do that